inbox/api/ns_api.py

```diff
@@ -300,7 +319,12 @@ def thread_api_update(public_id):
     data = request.get_json(force=True)
     if not isinstance(data, dict):
         raise InputError('Invalid request body')
-    update_thread(thread, data, g.db_session)
+
+    if g.api_features.optimistic_updates:
+        update_thread(thread, data, g.db_session, True)
```

Comment:
any reason we're not just passing `g.api_features.optimistic_updates` in as the argument to `update_thread()`. Seems kind of weird to put a conditional on a boolean and then explicitly pass in the exact same boolean when the same thing could be accomplished in a single line of code